diff --git a/configure b/configure
--- a/configure
+++ b/configure
@@ -6187,13 +6187,13 @@ case $target_os in
     win32|win64)
         disable symver
         LIBSUF=".lib"
-        if enabled shared; then
+#        if enabled shared; then
             # Cannot build both shared and static libs with MSVC or icl.
-            disable static
-            LIBSUF="-static.lib"
-        fi
+#            disable static
+#            LIBSUF="-static.lib"
+#        fi
         ! enabled small && test_cmd $windres --version && enable gnu_windres
         enabled x86_32 && check_ldflags -LARGEADDRESSAWARE
         add_cppflags -DWIN32_LEAN_AND_MEAN
         shlibdir_default="$bindir_default"
         LIBPREF=""
